chains/mainnet/Carbon.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  chainID: 'carbon-1',
+  lcd: 'https://api.carbon-network',
+  gasAdjustment: 1.75,
+  gasPrices: { swth: 100000000 },

```

Comment:
is this in "u" units?

Reply:
nope its without the u, our chain is a bit different